app/src/main/res/values/strings.xml

```diff
@@ -1675,7 +1675,13 @@
     <string name="menu_single_webview_open_external">Open in system browser</string>
     <!-- Donor experience -->
 
+    <!-- Places -->
     <string name="places_title">Places</string>
     <string name="places_permissions_denied">Location permission is required to see your location on the map. Please try again.</string>
+    <string name="places_link_preview_dialog_read_button">Read</string>
```

Comment:
These strings apply to the Link Preview component, so they shouldn't be mentioning "places" in their names.

Reply:
Done.